src/Microsoft.Extensions.RazorViews.Sources/AttributeValue.cs

```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Extensions.RazorViews
```

Comment:
Do we want these to all be in a `.Internal` namespace?

---
In reply to: [75024341](https://github.com/aspnet/Common/pull/139#discussion_r75024341) [](ancestors = 75024341)